# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/WebCodecsUtils.cpp
# Commit: e59a8dc5add5
# Full Hash: e59a8dc5add572e5ce65248019b1dab40c52f6d8
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - Implement an EncoderAgent to bridge Web Codecs and the PEM. r=chunmin
#   
#   Very similar to the decode side, except simpler because flushing's semantic
#   match.
#   
# ==============================================================================

diff -r a65dfea63fc7 -r e59a8dc5add5 dom/media/webcodecs/WebCodecsUtils.cpp
--- a/dom/media/webcodecs/WebCodecsUtils.cpp	Fri Dec 22 21:39:09 2023 +0000
+++ b/dom/media/webcodecs/WebCodecsUtils.cpp	Fri Dec 22 21:39:09 2023 +0000
@@ -508,3 +508,4 @@
       GetMediaThreadPool(MediaThreadType::PLATFORM_ENCODER),
       "WebCodecs encoding", false);
 }
+};  // namespace mozilla::dom
